src/Graphics/Mesh.cpp

```diff
@@ -89,14 +89,23 @@ void Mesh::Draw()
 
 void Mesh::Draw(uint32_t count, uint32_t startIndex)
 {
+	static const std::array<GLenum, 5> topologies = {
```

Comment:
appears to error on Windows:

```..\src\Graphics\Mesh.cpp(99): error C2676: binary '[': 'const std::array<GLenum,5>' does not define this operator or a conversion to a type acceptable to the predefined operator```

Also I'd imagine this could be a constexpr too.

Reply:
My bad it actually errors on the bit below: `auto topology = topologies[static_cast<size_t>(_topology)];`